01/ht/gendos379/01/script.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Makes calculations according to given attributes.
+ * All attributes should be given in separated parenthesis, e.g.:
+ * calc(3)('+')(2)
+ * @param {Number} a - First number for calculation
+ * @param {String} b - Arithmetic operator, possible values: '+' / '-' / '/' / '*'
+ * @param {Number} c - Second number for calculation
+ * @return {Number} - Result of calculation
+ */
+function calc(a) {
+  return function operate(b) {
+    if (b === '+') {
+      return function sum(c) {
+        return isNum(a) + isNum(c);
+      }
+    } else if (b === '-') {
+      return function subst(c) {
+        return isNum(a) - isNum(c);
+      }
+    } else if (b === '*') {
+      return function mult(c) {

```

Comment:
function name is redundant. No need to remove it, just keep in mind that you can skip it